examples/standalone_panorama/README.md

```diff
@@ -24,8 +24,7 @@ This is a non zonal deployment. The deployed infrastructure consists of:
   - a Network Security Group to give access to Panorama's public interface
 - a Panorama appliance with a public IP assigned to the management interface
 
-![standalone-panorama](https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/assets/2110772/b2dadd69-f5b5-4ac4-b356-467ef79cbb0b)
-
+![standalone-panorama](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/2110772/a2394f73-c0a8-4878-8693-825356abbd23)
```

Comment:
Here we will have conflict with #23 .

Moreover we should not have links for `vmseries` repo, rather `swfw`.

Reply:
Ahh my bad, this got automatically generated not sure why. I'm going to fix that. :+1:

Reply:
Fixed, thanks! :+1: